pkg/config/server/testdata/scylla-manager.yaml

```diff
@@ -46,3 +46,19 @@ repair:
   long_polling_timeout_seconds: 5
   age_max: 12h
   graceful_stop_timeout: 60s
+
+client:
+  scheme: http
+  timeout: 45s
```

Comment:
Isn't it 30s?

Reply:
This test is for setting a modified config, not the default values. The values in here are all arbitrary chosen for a test.